### DISCUSSION TITLE
Payment done but copilot no activated

### DISCUSSION THREAD

**Discussion by limordryer** [community/community#32867] [Copilot Conversations] [answered]
Hi,
I made yearly payment I can see in my profile all paid successfully, but still the vscode inform me every time the my subscription ended
Please help

**Comment by davecheney** [comment, marked as answer]
@limordryer do you have more than one GitHub account? 
<img width="475" alt="Screen Shot 2021-11-30 at 8 03 57 am" src="https://user-images.githubusercontent.com/7171/143942796-0c2b6d34-e0f4-46fe-b349-f26087b2c5ed.png">
Please try signing out, copilot will prompt you to sign back in.

**Comment by limordryer** [comment]
Thx working

**Comment by mqadi** [comment]
I do not recieve my paid sponsor ,I supposed to earn monthly which Pro plan
